airflow/ui/src/App.tsx

```diff
@@ -1,11 +1,61 @@
+/*!
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
 import { hot } from 'react-hot-loader';
 import React from 'react';
-import { Center, Heading } from '@chakra-ui/react';
+import { Route, Redirect, Switch } from 'react-router-dom';
+
+import Pipelines from 'views/Pipelines';
+import Pipeline from 'views/Pipeline';
+
+import EventLogs from 'views/Activity/EventLogs';
+
+import Config from 'views/Config';
+
+import Access from 'views/Access';
+import Users from 'views/Access/Users';
+import Roles from 'views/Access/Roles';
+
+import Docs from 'views/Docs';
+import NotFound from 'views/NotFound';
 
 const App = () => (
-  <Center height="100vh">
-    <Heading>Apache Airflow new UI</Heading>
-  </Center>
+  <Switch>
+    <Redirect exact path="/" to="/pipelines" />
+    <Route exact path="/pipelines" component={Pipelines} />
+    <Route exact path="/pipelines/:dagId" component={Pipeline} />
+
+    <Route exact path="/activity/event-logs" component={EventLogs} />
+
+    <Route exact path="/config" component={Config} />
+
+    <Route exact path="/access" component={Access} />
+    <Route exact path="/access/users" component={Users} />
+    <Route exact path="/access/users/new" component={Users} />
+    <Route exact path="/access/users/:id" component={Users} />
+    <Route exact path="/access/users/:id/edit" component={Users} />
+    <Route exact path="/access/roles" component={Roles} />
+
+    <Route exact path="/docs" component={Docs} />
```

Comment:
This was previously just a link out to docs -- do we need/want it to be a component now?

Reply:
Yes. Again, nothing set in stone, but we're thinking we can provide a better experience with [a page like we mocked up in the POC](https://nova.astronomer.io/docs).